sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -165,4 +154,222 @@ class ShowTablesSuite extends ShowTablesSuiteBase with CommandSuiteBase {
       }
     }
   }
+
+  test("show table extended in non-partitioned table") {
```

Comment:
shouldn't v2 have the same error?

Reply:
- v1 error class:  `_LEGACY_ERROR_TEMP_1251`
https://github.com/apache/spark/blob/e499ba80be6828dcf0c34d3f06f1cd12321ce5b8/common/utils/src/main/resources/error/error-classes.json#L4849-L4853

- v2 error class: `_LEGACY_ERROR_TEMP_1231`
https://github.com/apache/spark/blob/e499ba80be6828dcf0c34d3f06f1cd12321ce5b8/common/utils/src/main/resources/error/error-classes.json#L4804-L4808

Maybe we can merge it after this PR.